llvm/test/Transforms/InstCombine/cheri-memcpy-64bit-cap.ll

```diff
@@ -0,0 +1,84 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -instcombine -S | FileCheck %s
+target datalayout = "e-m:e-pf200:64:64:64:32-p:32:32-i64:64-n32-S128-A200-P200-G200"
+
+define ptr addrspace(200) @test_memcpy_cap(ptr addrspace(200) noundef %d, ptr addrspace(200) noundef %s) local_unnamed_addr addrspace(200) {
+; CHECK-LABEL: @test_memcpy_cap(
+; CHECK-NEXT:  entry:
+; CHECK-NEXT:    [[TMP0:%.*]] = load ptr addrspace(200), ptr addrspace(200) [[S:%.*]], align 8
+; CHECK-NEXT:    store ptr addrspace(200) [[TMP0]], ptr addrspace(200) [[D:%.*]], align 8
+; CHECK-NEXT:    ret ptr addrspace(200) [[D]]
+;
+entry:
+  ; memcpy can be transformed to capability load/store
```

Comment:
Use ;; for non-FileCheck comments, and put at the top of the test function rather than buried inside it

Reply:
Should be fixed now?